Library/ViewModels/SettingsAccountViewModel.swift

```diff
@@ -140,12 +156,17 @@ SettingsAccountViewModelOutputs, SettingsAccountViewModelType {
 
   public let dismissCurrencyPicker: Signal<Void, NoError>
   public let fetchAccountFieldsError: Signal<Void, NoError>
-  public let reloadData: Signal<(Currency, Bool), NoError>
+  public let reloadData: Signal<(Currency, Bool, Bool), NoError>
   public let presentCurrencyPicker: Signal<Currency, NoError>
   public let showAlert: Signal<(), NoError>
+  public let tableViewTopConstraint: Signal<CGFloat, NoError>
   public let transitionToViewController: Signal<UIViewController, NoError>
   public let updateCurrencyFailure: Signal<String, NoError>
 
   public var inputs: SettingsAccountViewModelInputs { return self }
   public var outputs: SettingsAccountViewModelOutputs { return self }
 }
+
+private func updateConstraintConstant(_ shouldHideEmailPasswordSection: Bool) -> CGFloat {
```

Comment:
If you update the number of sections in the data source, you shouldn't need to modify the header height 🤔.